basiclti/basiclti-oidc/src/java/org/sakaiproject/lti13/OIDCServlet.java

```diff
@@ -191,6 +191,7 @@ private void fancyRedirect(HttpServletRequest request, HttpServletResponse respo
 			PrintWriter out = null;
 			try {
 				out = response.getWriter();
+				response.setContentType("text/html");
```

Comment:
on line L#189 the cookie fetched directly .... can you just check for a session instead? manual cookie fetching is yuck

Reply:
The problem (I think) with checking the session is that we are trying to know *if* a session exists or if it needs to be restored.   If you just ask for a session and there is none - it will make a new session and log the user out.  So we need to know before we assume things work - whether or not they will work.  Make sense?